### ISSUE TITLE
Add Sherpa/Romeo Authority Controls to Journal Title field so that users can find policies by journal title

### ISSUE THREAD

**Issue by amandafrench** [VTUL/vtechworks#174]
See documentation at https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values

**Comment by alawvt** [comment]
Notes from https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values#AuthorityControlofMetadataValues-ConfigurationProperties,

In dspace.cfg,

un-comment [ChoiceAuthority Plugin](https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values#AuthorityControlofMetadataValues-ChoiceAuthorityPlugins) 

```
#plugin.named.org.dspace.content.authority.ChoiceAuthority = \
# org.dspace.content.authority.SHERPARoMEOJournalTitle = SRJournalTitle, \
```

[Select the choice plugin](https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values#AuthorityControlofMetadataValues-Selectingthechoiceplugin)

```
choices.plugin.dc.title.serial = SRJournalTitle
```

[Select the choice presentation style](https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values#AuthorityControlofMetadataValues-SelectingChoicePresentationStyle): select, suggest, or lookup

```
choices.presentation.dc.title.serial = select
```

I am not sure which one will be best, so we'll have to test them.

I believe we want [open choices](https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values#AuthorityControlofMetadataValues-OpenorClosedChoices)

```
choices.closed.dc.title.serial = false
```

I believe we want to [mark authority control](https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values#AuthorityControlofMetadataValues-MarkingAuthorityControl), but not [require an authority value](https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values#AuthorityControlofMetadataValues-RequiringAuthorityValue).
Let's try going with the default [minimum confidence level](https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values#AuthorityControlofMetadataValues-DefaultMinimumConfidence).

```
authority.minconfidence = ambiguous
```

See the [example field configurations](https://wiki.duraspace.org/display/DSPACE/Authority+Control+of+Metadata+Values#AuthorityControlofMetadataValues-Exampleofsomefieldconfigurations).

**Comment by amandafrench** [comment]
We need to wait on this until we've finished coming up with a plan for the default submission forms, as per #105, #186, #187, #188.

**Comment by alawvt** [comment]
TODO: check/develop associated hint for the lookup

**Comment by alawvt** [comment]
Resolved with PR #306.
